tools/resourceManagement/lib/resource/models/baseResource.js

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information. 
+
+'use strict';
+
+/**
+ * @class
+ * Initializes a new instance of the Resource class.
+ * @constructor
+ */
+class BaseResource {
+  constructor() { }
```

Comment:
I think this line is not necessary, if i remember correctly it does it implicitly.